### ISSUE TITLE
Cannot unlock Pro Edition under Linux KDE Plasma Desktop

### ISSUE THREAD

**Issue by Irratzo** [abulka/pynsource#104]
Hi there,

I just purchased Pynsource Pro Edition. Thank you for the product. However, I am unable to unlock the Pro Edition. In Pynsource, I click on Help > Purchase Pro Edition. No window opens. In the graphical terminal from which I started Pynsource, the following error message is printed.
```
kde-open5: /tmp/_MEIWprlhW/libz.so.1: version `ZLIB_1.2.9' not found (required by /usr/lib/x86_64-linux-gnu/libpng16.so.16)
```
For the same reason, I am also unable to report this as a bug.However, other subwindows (those from File, Edit etc.) are working, ie they open. There I get similar error messages in the terminal, but at least they are working. 
```
(Pynsource:670986): Gtk-WARNING **: Error loading theme icon 'image-missing' for stock: Unable to load image-loading module: /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufl
oader-png.so: /tmp/_MEIWprlhW/libz.so.1: version `ZLIB_1.2.9' not found (required by /usr/lib/x86_64-linux-gnu/libpng16.so.16)
```
How to resolve this issue?

Specs:

Pynsource installation: Binary from official website.
Pynsource version: 1.77

System: 
Operating System: Kubuntu 20.04
KDE Plasma Version: 5.18.5
KDE Frameworks Version: 5.68.0
Qt Version: 5.12.8
Kernel Version: 5.4.0-84-generic
OS Type: 64-bit
Processors: 4 × Intel® Core™ i5-3427U CPU @ 1.80GHz
Memory: 15.5 GiB of RAM

**Comment by abulka** [comment]
Hi Johannes,

Sorry you are having troubles with the rego. Ensure you are running the binary download not the snap, though you seem to have already done this. I'll test in the next few hours on my neon desktop vm - hopefully that will shed some light.

**Comment by Irratzo** [comment]
Thank you for the quick answer!

Update:

In the above message, I had used the wrong menu item (in the binary version). Looking again, I saw the menu item Help > Enter license. With that, I was able to register the pro version.

But now I cannot use the function File > Visualise Python Modules. That was the feature for which I purchased the Pro version in the first place.
The reason seems that the current binary version for Ubuntu I downloaded from the website today, version 1.77, doesn't have that menu item.
To test, I installed via snap instead. There I get version 1.8. There, the menu item File > Visualise Python Modules exists.
But, with the snap version I am unable to register the Pro version. When I click on Help > Enter license, I get this error message in the terminal:
```
Traceback (most recent call last):
 File "/snap/pynsource/19/pynsource-gui.py", line 1355, in OnEnterLicense
   from gui.settings import enter_license
ImportError: cannot import name 'enter_license'
```

So, I am back at square one, so to speak.
